caom2utils/caom2utils/tests/test_fits2caom2.py

```diff
@@ -1281,6 +1368,50 @@ def _get_headers(subject):
     return headers
 
 
+class TestResponse(object):
+
+    def __init__(self):
+        self.text = None
+
+    def close(self):
+        pass
+
+
+def _get_headers_2(external_url, timeout):
```

Comment:
Is this needed?

Reply:
It's gone, due to using 'with patch' from the previous comment.